### PR TITLE
Add logic to display the new ORA UI templates

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -** 
Note: the flags for whether to show the new UI in both python and JS both amount to "if True" for now, until a flag is baked in.

JIRA: [JIRA-XXXX](https://openedx.atlassian.net/browse/JIRA-XXXX)

**What changed?**

- [ More in depth breakdown of changes ]
- [ Peripheral things that got changed ]
- [ etc... ]

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

File: openassessment/templates/base.html [hunk 1]
```diff
@@ -1,3 +1,111 @@
...
+                'payload': {
+                    'url': 'http://localhost:1992/embedded/peer_assessment/23',
+                    'title': 'title',
+                    'height': '83vh',
+                }
+            },
+            document.referrer
+        );
+    }
+   */
```
Comment (nsprenkle) [inline openassessment/templates/base.html:109]: What is this for?

File: openassessment/xblock/openassessmentblock.py [hunk 2]
```diff
@@ -524,6 +524,46 @@ def get_student_item_dict(self, anonymous_user_id=None):
...
+            "title": self.title,
+            "prompts": self.prompts,
+            "prompts_type": self.prompts_type,
+            "rubric_assessments": ui_models,
+            "show_staff_area": self.is_course_staff and not self.in_studio_preview,
+        }
+        if any(model.name == "leaderboard" for model in ui_models):
+            context_dict["leaderboard_ui_model"] = [
+                model for model in ui_models if model.name == "leaderboard"
+            ][0]
```
Comment (nsprenkle) [inline openassessment/xblock/openassessmentblock.py:563]: Slightly different version that I think (1) reads better and (2) pops duplicate leaderboard from `ui_models`

```suggestion
        leaderboard_step = None

        for step_number, assessment_step in enumerate(ui_models):
            if step.name == "leaderboard"
                leaderboard_step = ui_models.pop(step_number)
                break
            
        context_dict = {
            "mfe_views": False,
            "title": self.title,
            "prompts": self.prompts,
            "prompts_type": self.prompts_type,
            "rubric_assessments": ui_models,
            "leaderboard_step": leaderboard_step,
            "show_staff_area": self.is_course_staff and not self.in_studio_preview,
        }

```

File: openassessment/templates/base.html [hunk 3]
```diff
@@ -1,3 +1,111 @@
...
+
+      <div class="openassessment problem" id="ora-view">
+          <iframe
+            title="title"
+            allow="microphone *; camera *; midi *; geolocation *; encrypted-media *, clipboard-write *"
+            frameborder="0"
+            src="http://localhost:1992/embedded/xblock/23"
+            style="width: 100%; height: 100%"
+          ></iframe>
+          {% if leadership_ui_model %}
```
Comment (nsprenkle) [inline openassessment/templates/base.html:59]: Typo?

```suggestion
          {% if leaderboard_ui_model %}
```

File: openassessment/xblock/openassessmentblock.py (hunk 2 above)
Comment (nsprenkle) [reply openassessment/xblock/openassessmentblock.py:563]: FWIW, my vote would also be to have a shared function to generate the context for these, but with `mfe_views` just forced off for studio.

**Review by nsprenkle — APPROVED** [review]
Looks good, with some nits on how to make a bit more readable and fix some typos.

File: openassessment/templates/base.html (hunk 1 above)
Comment (muselesscreator) [reply openassessment/templates/base.html:109]: ah, that is example of how to pull up the overlay modal until we have code to do so from inside the view.
